10_datastructures_lab/02.py

```diff
@@ -1,5 +1,21 @@
-"""
-Write a program that prints only unique values
-from sys.argv (without program name)
-"""
-
+from collections import Counter
+import sys
+
+anagrams = []
+#read all words from file
+with open(sys.argv[1], 'r') as f:
+    for line in f:
+       word = line[:-1]
+       wordCount = Counter(line[:-1])
+       
+       # see if same counter already appeared
+       found = False
+       for anagram in anagrams:
+           if anagram[0] == wordCount :
+               found = True
+               print "{} {}".format(word, anagram[1])
+
+        # Add to list if not found
+       if not found:
+            anagrams.append((wordCount, word))
+

```

Comment:
Python has an interesting construct to deal with such cases called the for...else. You can read about it here:
http://psung.blogspot.co.il/2007/12/for-else-in-python.html